.github/workflows/serenity-pr.yml

```diff
@@ -0,0 +1,73 @@
+name: Serenity - Assign Current Sprint to PR
+on:
+  pull_request:
+    types: [opened]
```

Comment:
Don't we want this to occur when the issue is _closed_? If we did this for every issue added to the board, we'd spend all our time updating issues.

If we have a re-opened issue event, we may want to _remove_ the sprint assignment, but I think we should hold off on that for the time being,

Reply:
I may have misunderstood the intention here. I thought the goal was to automatically tag new PRs with the current sprint? I haven't been looking at automatically tagging issues at this point.